Library/.rubocop.yml

```diff
@@ -100,6 +100,7 @@ Style/HashTransformValues:
 Style/HashAsLastArrayItem:
   Exclude:
     - 'Taps/*/*/{Formula/,}*.rb'
+    - '/**/{Formula/,}*.rb'
```

Comment:
```suggestion
    - 'Taps/*/*/*.rb'
    - '/**/Formula/*.rb'
```

Reply:
Good catch.

Reply:
No, this is only for (legacy) top-level formulae in taps.

Reply:
Gotcha.